sema/commons/j2/j2_functions.py

```
@@ -182,13 +292,17 @@ def xsd_format(content: Any, type_name: str, quote: str = "'") -> str:
+    # first try
     type_format_fn = XSD_FMT_TYPE_FN.get(type_name.lower(), None)
-    assert type_format_fn is not None, (
-        "type_name '%s' not supported." % type_name
-    )
+    if not type_name.startswith("auto"):
+        if not type_name.startswith("xsd:"):
+            type_name = "xsd:" + type_name
+
+        # second try
+        type_format_fn = XSD_FMT_TYPE_FN.get(type_name.lower(), None)
+        assert type_format_fn is not None, (
```

Ruff (0.8.2)

Use of `assert` detected

(S101)